src/test/default.xqy

```diff
@@ -111,14 +111,34 @@ declare function t:run-suite($suite as xs:string, $tests as xs:string*, $run-sui
       helper:log(text {"SUITE:", $suite}),
       try {
         helper:log(" - invoking suite setup"),
-        xdmp:invoke(fn:concat("suites/", $suite, "/suite-setup.xqy"))
+        xdmp:invoke(fn:concat("suites/", $suite, "/suite-setup.xqy")),
+        element t:test {
+          attribute name { "suite-setup.xqy" },
+          attribute time { functx:total-seconds-from-duration(xdmp:elapsed-time() - $start-time) },
+          element t:result {
+            attribute type {"success"}
+          }
+        }
       }
       catch($ex) {
         if ($ex/error:code = "XDMP-MODNOTFOUND" and
           fn:matches($ex/error:stack/error:frame[1]/error:uri/fn:string(), "/suite-setup.xqy$")) then
-          ()
+          element t:test {
+            attribute name { "suite-setup.xqy" },
+            attribute time { functx:total-seconds-from-duration(xdmp:elapsed-time() - $start-time) },
+            element t:result {
+              attribute type {"success"}
+            }
+          }
         else
-          (helper:log($ex), xdmp:rethrow())
```

Comment:
The rethrow() here prevented execution of the tests when suite-setup fails. I think that makes sense, since if the setup fails, that probably means the tests will fail too. The revised version here will go ahead and run the tests. I guess that doesn't really hurt anything, since the setup error will be at the top.